doc/api/n-api.md

```diff
@@ -4678,7 +4678,7 @@ napi_status napi_get_cb_info(napi_env env,
 * `[in-out] argc`: Specifies the length of the provided `argv` array and
   receives the actual count of arguments. `argc` can
   optionally be ignored by passing `NULL`.
-* `[out] argv`: Buffer to which the `napi_value` representing the arguments are
+* `[out] argv`: Array of `napi_values` to which the arguments will be
```

Comment:
Unless `napi_values` is a real type, it should not be in backticks. This document seems to use this instead:

```suggestion
* `[out] argv`: Array of `napi_value`s to which the arguments will be
```